src/pages/Main/Main.tsx

```diff
@@ -3,15 +3,16 @@ import { useLocation, useHistory } from "react-router";
 
 import { useTheme } from "@mui/material/styles";
 
+import { useSidebar } from "@store/sidebarContext";
 import { useComparison } from "@store/comparisonContext";
 import { useSelectedDistrict } from "@store/district/selectedContext";
-import { useSidebar } from "@store/sidebarContext";
 
 import Footer from "@components/Footer";
 import Header from "@components/Header";
 import Modal from "@components/Modal";
 import Sidebar from "@components/Sidebar";
 import CompatisonMode from "@components/ComparisonMode";
+import SideInformationsContainer from "@components/SideInformationsContainer";
```

Comment:
What do you this name `SideInformationsContainer` be like `SummaryCard`? @valdersonjr @enzoggqs

Reply:
I think it would be great.